templates/assets/cloudformation/elb.yml

```diff
@@ -210,21 +219,14 @@ Resources:
       Tags:
       - Key: Name
         Value: !Ref AWS::StackName
+      - !If [ShouldCreateElbAccessLogsS3Bucket, {"Key": 'mu:bucketpolicy', "Value": {"Ref": "ElbAccessLogsS3BucketPolicy"}}, !Ref 'AWS::NoValue']
       SecurityGroups:
       - !Ref ElbSG
       LoadBalancerAttributes:
-      - Key: access_logs.s3.enabled
-        Value: "true"
-      - Key: access_logs.s3.bucket
-        Value:
-          !If
-            - ShouldCreateElbAccessLogsS3Bucket
-            - Fn::Select:
-                - 0
-                - [!Ref ElbAccessLogsS3Bucket, !Ref ElbAccessLogsS3BucketPolicy]
-            - !Ref ElbAccessLogsS3BucketName
-      - Key: access_logs.s3.prefix
-        Value: !Ref ElbAccessLogsS3Prefix
+      - Key: 'access_logs.s3.enabled'
+        Value: !If [ShouldCreateElbAccessLogsS3Bucket, 'true', 'false']
+      - !If [ShouldCreateElbAccessLogsS3Bucket, {"Key": 'access_logs.s3.prefix', "Value": {"Ref": "ElbAccessLogsS3Prefix"}}, !Ref 'AWS::NoValue']
```

Comment:
Stylistically can you have this conditional match the higher line count format? It is easier to discern what is going on.